src/Catalyst.TestUtils/PeerSettingsHelper.cs

```diff
@@ -41,8 +41,8 @@ public static IPeerSettings TestPeerSettings(byte[] publicKey = default, int por
             peerSettings.NetworkType.Returns(NetworkType.Devnet);
             peerSettings.PublicKey.Returns(finalPublicKey.KeyToString());
             peerSettings.Port.Returns(port);
-            peerSettings.PayoutAddress.Returns("my_pay_out_address");
             peerSettings.BindAddress.Returns(IPAddress.Loopback);
+            peerSettings.PublicIpAddress.Returns(IPAddress.Loopback);	
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace all tab characters in this file by sequences of white-spaces.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4366149)